app/main.py

```diff
@@ -1,5 +1,63 @@
+from __future__ import annotations
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
```

Comment:
You must specify inner type of list
```suggestion
    def __init__(self, name: str, weight: int, coords: list[int] | None = None) -> None:
```